pages/apim/3.x/installation-guide/docker/installation-guide-docker-compose.adoc

```diff
@@ -1,32 +1,26 @@
-= Docker Compose Installation
+= Custom Install with Docker Compose
 :page-sidebar: apim_3_x_sidebar
 :page-permalink: apim/3.x/apim_installation_guide_docker_compose.html
 :page-folder: apim/installation-guide/docker
 :page-layout: apim3x
 :page-description: Gravitee.io API Management - Installation Guide - Docker - Compose
 :page-keywords: Gravitee.io, API Management, apim, guide, manual, docker, compose, linux
-:page-toc: false
 :page-liquid:
 :table-caption!:
 
-// author: Tom Geudens
+This page describes how to install and run APIM in Docker containers on `localhost` using the `docker compose` command and a specified filesystem for persistence and plugins. 

```

Comment:
Might want to modify slightly as the quickstart method still provides persistence through volumes. Maybe something along the lines of "more granular control of where persistence data is stored and the ability to add custom plugins".

Reply:
Good idea. Changed throughout.